client/src/components/Home/index.tsx

```diff
@@ -213,6 +257,44 @@ export default class Home extends React.Component<any, IHomeState> {
     return (
       <div>
         <h1>Bot Analytics</h1>
+        
+      <Button
+        icon
+        tooltipLabel="Import dashboard"
+        onClick={this.onOpenImport.bind(this)}
+        component={Link}
+        label="Import dashboard"
+      >file_upload
+      </Button>,
+    <Dialog
+      id="ImportDashboard"
+      visible={importVisible}
+      title="Import dashboard"
+      dialogStyle={{ width: '50%' }}
+      modal
+      actions={[
+        { onClick: this.onCloseImport, primary: false, label: 'Cancel' },
+        { onClick: this.onSubmitImport, primary: true, label: 'Submit', disabled: !file },
+      ]}
+    >
+      <FileUpload
+        id="dashboardDefenitionFile"
+        primary
+        label="Choose File"
+        accept="application/json"
```

Comment:
Correct! thanks. indeed the import expects .js file and not .json. I have incorporated this change
(and yes both export and import use the same format, the .js file format)